squareconnect/api_client.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, host=None, header_name=None, header_value=None, cookie=None):
             self.host = host
         self.cookie = cookie
         # Set default User-Agent.
-        self.user_agent = 'Square-Connect-Python/2.0.2'
+        self.user_agent = 'Swagger-Codegen/2.0.2/python'
```

Comment:
oh wait, this is because we don't explicitly set our user agent in configuration. We should probably add a line like  `UserAgent: "Square-Connect-Language/1.0.0"` to each of our configuration files so that we can keep a handle on them.